schedule.yaml

```diff
@@ -32,12 +32,6 @@ tabs:
     label: snowex
     leads:
     - H.P. Marshall
-  - title: 'Tutorial: ICESat-2 Mission Overview (Room 345)'
-    description: All about ICESat-2 and the data products
-    time: 11:15 - 12:00
-    label: icesat-2
-    leads:
-    - Tyler Sutterley
```

Comment:
Should this be removed?

Reply:
my bad, if it's late and you mistype a label (ICESat-2) those events get deleted